client/post-editor/editor-sidebar/index.jsx

```diff
@@ -49,6 +52,16 @@ export default React.createClass( {
 						selectedId={ this.props.post ? this.props.post.ID : null }
 					/>
 					: <div>
+					<EditorActionBar
+						isNew={ this.props.isNew }
+						onTrashingPost={ this.props.onTrashingPost }
+						onPrivatePublish={ this.props.onPublish }
+						post={ this.props.post }
+						savedPost={ this.props.savedPost }
+						site={ this.props.site }
+						type={ this.props.type }
+					/>
+					<EditorNotice { ...this.props.notice } />
```

Comment:
The least maintenance option is clearly to have it show only in the main tab. I've changed it.